lib/cylc/cfgspec/suite.py

```diff
@@ -352,6 +354,16 @@ def coerce_interval_list(value, keys, args, back_comp_unit_factor=1,
                 'execution timeout'           : vdr( vtype='interval_minutes'),
                 'reset timer'                 : vdr( vtype='boolean', default=False ),
                 },
+            'events' : {
+                'handler'                     : vdr( vtype='string_list', default=[] ),
+                'handler events'              : vdr( vtype='string_list', default=[] ),
+                'handler retry delays'       : vdr( vtype='interval_minutes_list', default=[] ),
+                'mail events'                 : vdr( vtype='string_list', default=[] ),
+                'mail from'                   : vdr( vtype='string' ),
+                'mail retry delays'           : vdr( vtype='interval_minutes_list', default=[] ),
+                'mail smtp'                   : vdr( vtype='string' ),
+                'mail to'                     : vdr( vtype='string' ),
+                },
```

Comment:
Can you add setting of default values for "handlers" and "handler events" as has been done recently for hooks under #1501